### PR TITLE
More design tweaks that came up in today's meeting

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

File: Research/Research.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1025,6 +1025,10 @@
 		FF8B54EC1FCE6D15006B6937 /* RSDColor.swift in Sources */ = {isa = PBXBuildFile; fileRef = FFDFF5CE1FC3EE5C009713E8 /* RSDColor.swift */; };
 		FF8B54ED1FCE6D15006B6937 /* RSDExceptionHandler.h in Headers */ = {isa = PBXBuildFile; fileRef = FFF159DE1FB4D7A60061BA93 /* RSDExceptionHandler.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		FF8B54EE1FCE6D15006B6937 /* RSDExceptionHandler.m in Sources */ = {isa = PBXBuildFile; fileRef = FFF159DF1FB4D7A60061BA93 /* RSDExceptionHandler.m */; };
+		FFABCB9422D9275600D11109 /* RSDCountdownUIStepObject.swift in Sources */ = {isa = PBXBuildFile; fileRef = FFABCB9322D9275600D11109 /* RSDCountdownUIStepObject.swift */; };
```
Comment (Erin-Mounts) [inline Research/Research.xcodeproj/project.pbxproj:1028]: are there supposed to be four instances of the same file in here? or is Xcode just stuttering

File: Research/Research/RSDActiveUIStepObject.swift [hunk 2]
```diff
@@ -58,6 +58,7 @@ open class RSDActiveUIStepObject : RSDUIStepObject, RSDActiveUIStep {
     /// Should the task end early if the task is interrupted by a phone call? Default = `false`.
     open var shouldEndOnInterrupt : Bool = false
     
+
```
Comment (Erin-Mounts) [inline Research/Research/RSDActiveUIStepObject.swift]: extra whitespace

File: Research/Research/RSDCountdownUIStepObject.swift [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    
+    /// The active step that follows this step.
+    public internal(set) var activeStep: RSDActiveUIStep?
+    
+    /// The default step type is `countdown`.
+    open override class func defaultType() -> RSDStepType {
+        return .countdown
+    }
+    
+    /// Override setting the title to return a value of "Begin in.." if the title is `nil`.
```
Comment (Erin-Mounts) [inline Research/Research/RSDCountdownUIStepObject.swift]:
```suggestion
    /// Override setting the title to return a value of "Begin in..." if the title is `nil`.
```

**Review by Erin-Mounts — APPROVED** [review]
some nits, looks good

File: Research/Research.xcodeproj/project.pbxproj (hunk 1 above)
Comment (syoung-smallwisdom) [reply Research/Research.xcodeproj/project.pbxproj:1028]: One for each target: iOS, tvOS, watchOS, macOS

Comment (Erin-Mounts) [reply Research/Research.xcodeproj/project.pbxproj:1028]: ah, now it makes sense. :+1:

**Pull request merged**